lib/imgix/rails/picture_tag.rb

```diff
@@ -4,25 +4,32 @@
 class Imgix::Rails::PictureTag < Imgix::Rails::Tag
   include ActionView::Context
 
-  def initialize(source, options, default_options, breakpoints)
+  def initialize(source, tag_options: {}, url_params: {}, breakpoints: {}, widths: [])
```

Comment:
I'm actually not a fan of `tag_options` as a name here. @frederickfogerty and I were recently discussing a similar change he's making on the react-imgix and we settled on `html_attributes`. I think it's more precise and would like this to be consistent across relevant libraries.

Reply:
Nope, sounds like you've got some solid thought behind it. Go with your gut.